### ISSUE TITLE
Cannot Login After Installation

### ISSUE THREAD

**Issue by theshades** [RestyaPlatform/board#1177]
Greetings yall!
I have done a full install, the nginx error log is clean, etc. but when I try and login with my own account or with the admin account, I get the following error:
`` Sorry, login failed. Either your username or password are incorrect or admin deactivated your account.``
I completed a database check and say both accounts are enabled (isactive and isemailactive are both = t).

Any ideas? Not saying this isn't my fault again, but trying to get this online ASAP!

Thanks,

Shades

**Comment by christianschupp** [comment]
Hey theshades,

i latch into your issue as i can't login either. The login page is showing up fine, but neither the default "user" nor "admin" account works. I then created my own account. The activation mail was sent and the account was successfully activated through the mail. But login is not possible giving me the same error "Sorry, login failed. Either your username or password are incorrect or admin deactivated your account."

I've used the restyaboard.conf for nginx, nothing modified. The system is clean (dedicated server), nothing else running on it except the requirements: nginx, php7.0-fpm, postgresql. I'm using Ubuntu 16.04LTS (64-bit). The API responds just fine with JSON. No 404 or anything else. Error.logs are clean, Database seems fine too. Already tried to enable/disable debug mode in config.inc.php. DB user "restya" has all privileges granted (is owner of restyadb, has privileges on tables and sequences, all operations are allowed).

I just don't come to a solution. Sadly, this is pretty exhausting.

- Chris
